CRM/Activity/Form/Activity.php

```diff
@@ -823,7 +823,7 @@ public function buildQuickForm() {
    */
   public static function formRule($fields, $files, $self) {
     // skip form rule if deleting
-    if (CRM_Utils_Array::value('_qf_Activity_next_', $fields) == 'Delete') {
+    if ($fields['_qf_Activity_next_'] ?? NULL === 'Delete') {
```

Comment:
This seems a little ambiguous about which operator goes first (?? or ===)
```suggestion
    if (($fields['_qf_Activity_next_'] ?? NULL) === 'Delete') {
```

Reply:
lol -  I was going the brackets but  thought  you  have been removing them from lots of places where I wanted them